math/physics/src/TLorentzVector.cxx

```diff
@@ -5,9 +5,22 @@
 //    Oct 20 1999: dito in Double_t operator()
 //    Jan 25 2000: implemented as (fP,fE) instead of (fX,fY,fZ,fE)
 
-
 /** \class TLorentzVector
     \ingroup Physics
+
+## Disclaimer
+In order to represent 4-vectors, TLorentzVector shall not be used.
+ROOT provides specialisations of the ROOT::Math::LorentzVector template which
+are superior from the runtime performance offered, i.e.:
+  - ROOT::Math::XYZTVector vector based on x,y,z,t coordinates (cartesian) in double precision
```

Comment:
Maybe remove dupe "vector" and use code style:
```
`ROOT::Math::XYZTVector` based on x,y,z,t coordinates (Cartesian) in double precision
```

Reply:
note for myself(?): this needs to be changed in the doc of GenVector too.